game/src/Game/Autoload/CursorManager.gd

```diff
@@ -0,0 +1,173 @@
+extends Node
+
+class CompatCursor:
+	#cursor properties
+	var cursor_name : String
```

Comment:
```suggestion
	var cursor_name : StringName
```
Also change the argument type in `_init` and `set_compat_cursor` to `StringName`